Assets/Scripts/Map/MapUtil.cs

```diff
@@ -819,6 +819,30 @@ public static Vector MakeVector(int VID, int PID, double Hang, double Vang)
             }
         }
 
+        public struct Node
+        {
+            public int NID;
+            public int PID;
+
+            public static Node GetDefaultNode()
+            {
+                return new Node()
+                {
+                    NID = 1,
+                    PID = 1,
+                };
+            }
+
+            public static Node MakeNode(int NID, int PID)
+            {
+                return new Node()
+                {
+                    NID = NID,
```

Comment:
Are the parameters shadowing the Node fields NID/PID?

Reply:
> @hakuturu583 can you sign the CLA? Thanks.
Excuse me, what is CLA?